### ISSUE TITLE
2025-11-07: sgl-project/sglang 仓库更新报告

### ISSUE THREAD

**Issue by yan97ao** [yan97ao/argus#401]
# 每日更新报告（2025-11-07）

## sgl-project/sglang

| 提交时间 | 作者 | 提交信息 |
|----------|------|----------|
| 2025-11-07 21:30:04 | Yang Kaiyong | optm(checkpoint-engine): disable multi-thread loading when update weights (#12374) |
| 2025-11-07 21:27:43 | Chen1022 | Fix missing dp_max_padding argument in set_dp_buffer_len (#12812) |
| 2025-11-07 17:34:04 | hlu1 | [Deepseek V3.2] Only skip Indexer logits computation when is_extend_without_speculative (#12816) |
| 2025-11-07 16:34:46 | Jonah Bernard | [MoE] Add Comprehensive MoE Integration Tests (#12090) |
| 2025-11-07 13:59:57 | elvischenv | [Bugfix] Fix illegal memory access (#12758) |
| 2025-11-07 13:28:43 | Johnsonms | [Test] Add DeepSeekV3.2 NSA Indexer Test Suite (#12520) |
| 2025-11-07 13:19:49 | Yuhao Yang | fix multimodal gen issues (#12765) |
| 2025-11-07 11:38:35 | Kalyan Kumar | fixes hardcoded "cuda" device references in unit tests to use a dynamic device selection (#12761) |
| 2025-11-07 11:14:27 | Tianyu Guo | Add PP support for dots_vlm (#12763) |
| 2025-11-07 10:58:44 | hzh0425 | [HotFix]: Add missing SGLANG_EPLB_HEATMAP_COLLECTION_INTERVAL env var (#12776) |
| 2025-11-07 10:48:03 | hzh0425 | [hotfix]: Resolve ModuleNotFoundError in PD deployment for is_in_ci() (#12772) |
| 2025-11-07 09:26:08 | YAMY | [DeepSeek-V3.2][NSA] Enable MHA Pathway for Short Sequence Prefill on B200 (SM100) (#12788) |
| 2025-11-07 07:51:44 | Gaurav Verma | [AMD] Update wave-lang to 3.8.2 (#12576) |
| 2025-11-07 07:27:28 | Rain Jiang | ignore the deepgemm check when the model weight with nvfp4 and moe ba… (#12782) |
| 2025-11-07 06:27:40 | alisonshao | Add bot-bump-kernel-version-to-sglang workflow (#12794) |
| 2025-11-07 06:06:22 | Nicolas Castet | Fix hang in deepgemm compilation with symmetric memory enabled (#12715) |
| 2025-11-07 05:11:28 | Yineng Zhang | chore: update CODEOWNERS (#12795) |
| 2025-11-07 03:53:58 | Lee Nau | [fix] Only enable flashinfer all reduce fusion by default for single-node servers (#12724) |
| 2025-11-07 03:33:31 | R0CKSTAR | Pin uvloop to 0.21.0 (#12279) |
| 2025-11-07 03:25:49 | Rahul Chalamala | [docs] Remove redundant --disable-radix-cache option from  (#12717) |
| 2025-11-07 03:24:48 | jianan-gu | [CPU] Fix TP padding case with weight block size (#8243) |
| 2025-11-07 03:23:12 | Jimmy | [Fix]HTTP Stream raise exception (#11904) |
| 2025-11-07 03:05:42 | Teng Ma | [mem pool] bugfix: wrong position for self.device in Mamba (#12684) |
| 2025-11-07 02:49:32 | Mattheliu | [Docs] fix dead links in multiple documentation pages (#12764) |
| 2025-11-07 02:34:23 | gaopengff | [Intel XPU]Update pytorch xpu to 2.9 (#12363) |
| 2025-11-07 02:33:41 | gaopengff | [Intel]Add 'intel_xpu' attention backend for llama4 (#11051) |
| 2025-11-07 02:20:00 | Chang Su | [router][grpc] Add more mcp test cases to responses api (#12749) |
| 2025-11-07 00:46:19 | sglang-bot | chore: bump SGLang version to 0.5.5 (#12739) |
